build/vega-lite-schema.json

```diff
@@ -4020,6 +3678,56 @@
       ],
       "type": "string"
     },
+    "ErrorExtraEncoding": {
```

Comment:
This is actually never referred anywhere. Probably an example of the bug in the generator.

Reply:
Ok. I re-declare this in a way that this problem would no longer surface.

Reply:
Yeah, we sometimes output unnecessary stuff but it should not hurt (except for larger schemas).

Reply:
lol. Isn't Vega-Lite schema large enough?

Reply:
Oh, I'm not saying it's not a bug but it's not a blocking bug.